QuizApp/QuizListCollectionViewCell.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import UIKit
+import SnapKit
+import Kingfisher
+
+class QuizListCollectionViewCell: UICollectionViewCell {
+
+    static let id = "QuizListCollectionViewCell"
+
+    private var quiz: QuizModel!
+    private var quizImageView: UIImageView!
+    private var quiztitleLabel: UILabel!
+    private var descriptionLabel: UILabel!
+    private var difficultyView1: UIView!
+    private var difficultyView2: UIView!
+    private var difficultyView3: UIView!
+    private var difficultyStackView: UIStackView!
```

Comment:
Extract this to a separate view e.g. DifficultyView, that would be able to configure itself based on the difficulty level that you pass to it.